### ISSUE TITLE
Additional filters for service list endpoint

### ISSUE THREAD

**Issue by sabedevops** [openziti/edge#1154]
Please consider adding tags to the set of service list filters. In most other domains, tags are primary used for filtering. Current set is below. 

`GET /edge/management/v1/services`

```json
"filterableFields": [
    "id",
    "createdAt",
    "updatedAt",
    "isSystem",
    "name",
    "terminatorStrategy",
    "roleAttributes"
]
```
It would also be useful to be able to filter by a service's configs. There are cases where we want to know if a service supplies a config of a certain type.

**Comment by plorenz** [comment]
filtering by tags will be fixed by storage#21

**Comment by plorenz** [comment]
Once the associated PR goes through, you can search for services with associated circuits using the following syntax:

`anyOf(configs) = "<some config id>"`
